src/bundler/bundle_v2.zig

```diff
@@ -1717,11 +1717,16 @@ pub const BundleV2 = struct {
 
             this.poll_ref.unref(globalThis.bunVM());
             const promise = this.promise.swap();
-            const root_obj = JSC.JSValue.createEmptyObject(globalThis, 2);
 
             switch (this.result) {
                 .pending => unreachable,
-                .err => {
+                .err => brk: {
+                    if (this.config.throw_on_error) {
+                        promise.reject(globalThis, this.log.toJSAggregateError(globalThis, bun.String.static("Bundle failed")));
```

Comment:
Consider using a more descriptive message than "Bundle failed".

Reply:
this message is ok because the `.errors` property of this aggregate error contains the actual errors.

what is your suggestion?